### PR TITLE
add documentation for copy-settings cmdlet

### PR DESCRIPTION
**Pull request by erik0686**
The new cmdlet "Copy-UserinternatioanlSettingsToSystem" has recently been created for PowerShell so I'm creating the documentation for it. 
This cmdlet is still not shipped, but I think creating the PR with time will be of great benefit 👍 

This cmdlet copies the current user's international settings (Display language, Input language, Format/locale and Location/GeoID) to one or both of the following:
    * Welcome screen and system accounts
    * New user accounts
Depending on which parameters are set to true.

### CODE REVIEW

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (jefgen) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:25]: I think we should likely add a note that the user must be an admin in order for the operation to succeed. 
For example, on the `Set-WinSystemLocale` cmdlet [here](https://docs.microsoft.com/en-us/powershell/module/international/set-winsystemlocale?view=win10-ps), they use this expression:
> This is a system setting. It can only be changed by a user who has Administrator permissions. Changes take effect after the computer is restarted.

I think it might make sense to use the same wording. :)

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (jefgen) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:16]: Hm, this makes me think we might need to bump the version number.

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (jefgen) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:41]:
```suggestion
When any of the parameters is set to $True, this gets the current values from the current user and copies them to the system settings for the selected options based on the parameters that were set to $True.
```

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (erik0686) [reply docset/windows/international/Copy-UserInternationalSettingsToSystem.md:25]: That's a very good point :D Thanks Jeff :)

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (jefgen) [reply docset/windows/international/Copy-UserInternationalSettingsToSystem.md:41]: lets not mention the registry, as the cmdlet doesn't actually read from the registry.
(it uses your new API).

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (jefgen) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:46]:
```suggestion
### Example 1: Copy settings into both the Welcome screen and new user accounts
```

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (jefgen) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:51]:
```suggestion
This example copies the Windows Display language, Input language, Regional Format/locale and Location/GeoID into both the Welcome screen and the new user accounts.
```

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (jefgen) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:25]:
```suggestion
Copies the current user's international settings (Windows Display language, Input language, Regional Format/locale and Location/GeoID) to one or both of the following:
```

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (jefgen) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:59]:
```suggestion
This example copies the Windows Display language, Input language, Format/locale and Regional Location/GeoID to the Welcome screen only.
```

**Comment by e0i** [comment]
Closing and re-opening to refresh validations.

**Comment by e0i** [comment]
cc @JohanFreelancer9 for comprehensive copy editing. Thanks

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (get-itips) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:34]:
```suggestion
Copy-UserInternationalSettingsToSystem [-WelcomeScreen <Boolean>] [-NewUser <Boolean>] [<CommonParameters>]
```

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (get-itips) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:6]: This should be a github username.

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (dariomws) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:39]:
```suggestion
The **Copy-UserInternationalSettingsToSystem** cmdlet gets the **international settings** from the current user.
You can decide whether to copy these settings into the **Welcome screen and system accounts**, the **New user accounts** or both.
```

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (dariomws) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:67]:
```suggestion
Copies the settings into the Welcome Screen and System accounts.
```

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (dariomws) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:79]:
```suggestion
Copies the settings into the new user accounts.
```

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (dariomws) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:91]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by dariomws — APPROVED** [review]
I approve and suggest some small fixes. Thanks!

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (JohanFreelancer9) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:39]:
```suggestion
You can decide whether to copy these settings into the **Welcome screen and system accounts**, the **New user accounts**, or both.
```

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (JohanFreelancer9) [inline docset/windows/international/Copy-UserInternationalSettingsToSystem.md:67]:
```suggestion
Copies the settings into the Welcome Screen and system accounts.
```

File: docset/windows/international/International.md
Comment (JohanFreelancer9) [inline docset/windows/international/International.md:85]:
```suggestion
Copies the current user's international settings to the Welcome screen and system accounts and/or new user accounts.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few minor edits, please.

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/windows/international/Copy-UserInternationalSettingsToSystem.md
Comment (e0i) [reply docset/windows/international/Copy-UserInternationalSettingsToSystem.md:6]: @erik0686 The PR is good to go after applying the above suggestion. Thank you for your contributions.

**Review by get-itips — APPROVED** [review]

**Comment by erik0686** [comment]
Thank you all for your feedback, help and time!

**Comment by tiburd** [comment]
Hi, Erik/@erik0686, 

**Following are blocking issues that need to be resolved before we can merge:**

docset/windows/international/Copy-UserInternationalSettingsToSystem.md
- After correcting the metadata (the three hyphens at the beginning of the file need to be on line 1), there's now a warning regarding the metadata: 
   `Invalid value for 'ms.technology': 'powershell-windows' is not valid with 'ms.prod' value 'w10'.`
- Lines 25, 26: Bullets aren't formatted correctly.

***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master)

**Blocking**

- The validation status must be "Passed."
- The content preview must be clean on staging. It cannot contain any obvious formatting issues.

***
Thanks!

**Comment by erik0686** [comment]
Thank you @tiburd!

I have fixed this :) I left the ms.technology blank, since this is how the rest of the cmdlets have it. If there is something you would like me to put there instead, I'll be happy to do it :)

Thanks!

**Comment by tiburd** [comment]
Hi, Erik/@erik0686, 

Thank you for fixing the blocking issues. Merging this PR now....

**Comment by jefgen** [comment]
Regarding this point from the original PR description:

> This cmdlet is still not shipped

This documentation page is for a new cmdlet that will ship in a future or upcoming version of Windows.

Is there some way to flag or denote in the documentation that this cmdlet is only present on upcoming versions of Windows? (Perhaps, something like version `xxx` or greater?).
Or alternatively, could this be noted as preliminary documentation for an upcoming new cmdlet?

Thanks!

**Comment by dariomws** [comment]
Hi @jefgen,

To give you an example, for Teams cmdlets in preview, a note is added at the beginning of the article:
https://docs.microsoft.com/powershell/module/teams/grant-csgrouppolicypackageassignment

**Comment by jefgen** [comment]
> To give you an example, for Teams cmdlets in preview, a note is added at the beginning of the article:
> https://docs.microsoft.com/powershell/module/teams/grant-csgrouppolicypackageassignment

Interesting -- Thanks!

**Pull request merged**
